test_runner/regress/test_broken_timeline.py

```diff
@@ -114,6 +114,7 @@ def test_timeline_init_break_before_checkpoint(neon_env_builder: NeonEnvBuilder)
         [
             ".*Failed to process timeline dir contents.*Timeline has no ancestor and no layer files.*",
             ".*Timeline got dropped without initializing, cleaning its files.*",
+            ".*Failed to load index_part from remote storage, failed creation?.*",
```

Comment:
Filter instead so that the existence of initdb does not imply existence of index_part.json.

Reply:
@arpad-m Checked, list prefixes is used so this error case will cost us one `get_object -> NotFound` on start, but the warning is actionable, should this ever happen.

It should be printed at most for once because control plane should retry the operation failure caused by pageserver restart AND after that retry the timeline will be found and will be overwritten with new tar.zst and index_part.json and now work.